package.json

```diff
@@ -13,27 +13,30 @@
   },
   "dependencies": {
     "aws-sdk": "^2.1.49",
-    "mime": "^1.3.4"
+    "mime": "^2.3.1"
```

Comment:
Forgot about this when I did the initial review, but it might be worth adding the `engines` property with the minimum required node version

Reply:
Will do

Reply:
Added